srcpkgs/wee-slack/template

```diff
@@ -3,7 +3,7 @@ pkgname=wee-slack
 version=2.3.0
 revision=1
```

Comment:
Please increase the revision so the builders will pick up the change in the template and rebuild the package.

CC @renatoaguiar

Reply:
Sorry, forgot that.